### PR TITLE
Swap react-portal with react-overlays while maintaining API

### PR DESCRIPTION
**Pull request by PeteJodo**
#### Todo
- [x] examples working
- [ ] update tests
- [ ] update docs

### CODE REVIEW

**Comment by PeteJodo** [comment]
The PR I was talking about. Still need to fix tests and update the `README` but I can do that after some discussion. No rush on this.

File: lib/FlexModalWrapper.css [hunk 1]
```diff
@@ -5,6 +19,11 @@
     max-width: 100%;
     max-height: 100%;
     overflow-y: auto;
+    z-index: 9999;
```
Comment (antialias) [inline lib/FlexModalWrapper.css]: Can you make this a variable with postcss?

Comment (PeteJodo) [inline lib/FlexModalWrapper.css]: yeah sure. I'll actually organize the css a little better. I didn't really pay too much attention to it really

File: lib/FlexModalWrapper.css [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-align: center;
+    box-pack: center;
+    z-index: 9900;
```
Comment (antialias) [inline lib/FlexModalWrapper.css]: this one too

File: lib/FlexModalWrapper.css [hunk 3]
```diff
@@ -13,14 +32,6 @@
...
-    align-items: center;
-    justify-content: center;
-    box-align: center;
-    box-pack: center;
-    z-index: 9900;
-}
-
-:local(.overlayStyle) {
     background: rgba(0,0,0,0.2);
+    z-index: 9900;
```
Comment (antialias) [inline lib/FlexModalWrapper.css]: and this one

**Comment by antialias** [comment]
Assigning back to @PeteJodo since this has been hanging out in my "assigned" list for over a month.

**Pull request closed without merging**